WalletWasabi.Fluent/ViewModels/Wallets/Send/ChangeAvoidanceSuggestionViewModel.cs

```diff
@@ -75,6 +75,11 @@ public partial class ChangeAvoidanceSuggestionViewModel : SuggestionViewModel
 	public static async Task<IEnumerable<ChangeAvoidanceSuggestionViewModel>> GenerateSuggestionsAsync(
 		TransactionInfo transactionInfo, Wallet wallet, BuildTransactionResult requestedTransaction)
 	{
+		if (transactionInfo.Address is null)
```

Comment:
@soosr need your help on this one, should be avoidable.

Reply:
discussed with @soosr we will tackle this in a different PR in future, for now we shall throw a more appropriate exception.

Reply:
refactored to avoid this possibility completly.